src/lib/multiply/README.md

```diff
@@ -0,0 +1,97 @@
+# Multiply
+
+`multiply` is a generic plugin for multiplying two or more values in an `input` array.
+
+You provide the names of the values you want to multiply, and a name to use to append the product to the output array.
+
+For example, you could multiply `cpu/energy` and `network/energy` and name the result `energy-product`. `energy-product` would then be added to every observation in your input array as the product of `cpu/energy` and `network/energy`.
+
+## Plugin name
+
+IF recognizes the plugin as `multiply`
```

Comment:
if doesn't have anything to do with naming, name is custom thing in if

Reply:
@jmcook1186 should I remove the `Plugin name` section from all readme files?